Bottom: 08544ae7b8be07a8e548d12ef578a3fed24c1400
Top:    01bfff2ed951566c1f6affd1554f66db078e8b30
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-01 16:04:55 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
